packages/native/src/polywrap_native.udl

```diff
@@ -33,7 +33,7 @@ interface FFIInvoker {
   [Throws=FFIError]
   sequence<FFIUri> get_implementations(FFIUri uri);
 
-  record<DOMString, sequence<FFIUri>>? get_interfaces();
+  record<FFIUri, sequence<FFIUri>>? get_interfaces();
```

Comment:
iirc @krisbitney has some concerns with this? can u confirm is i am wrong please; we can approve and merge this PR

Reply:
No, it's fine with me